build_resolvers/test/flutter_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Fast fail the script on failures.
+set -ex
+
+# Get Flutter.
+echo "Cloning master Flutter branch"
+git clone https://github.com/flutter/flutter.git ./flutter
+
+./flutter/bin/cache/dart-sdk/bin/dart test/resolver_test
```

Comment:
I thought `flutter/bin/cache` didn't exist until after some setup step - is this working for you?

Reply:
Nope - still working through this.

I grabbed these travis setup steps from the devtools repo but they must be doing some additional stuff later on.